openmetadata-ui/src/main/resources/ui/src/pages/RolesPage/RolesDetailPage/RolesDetailPage.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Card, Col, Empty, Row, Table, Tabs } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { AxiosError } from 'axios';
+import { isEmpty, uniqueId } from 'lodash';
+import { EntityReference } from 'Models';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { getRoleByName } from '../../../axiosAPIs/rolesAPIV1';
+import Description from '../../../components/common/description/Description';
+import RichTextEditorPreviewer from '../../../components/common/rich-text-editor/RichTextEditorPreviewer';
+import TitleBreadcrumb from '../../../components/common/title-breadcrumb/title-breadcrumb.component';
+import Loader from '../../../components/Loader/Loader';
+import {
+  GlobalSettingOptions,
+  GlobalSettingsMenuCategory,
+} from '../../../constants/globalSettings.constants';
+import { Role } from '../../../generated/entity/teams/role';
+import { getEntityName } from '../../../utils/CommonUtils';
+import {
+  getPolicyWithFqnPath,
+  getSettingPath,
+} from '../../../utils/RouterUtils';
+import { showErrorToast } from '../../../utils/ToastUtils';
+import './RolesDetail.less';
+
+const { TabPane } = Tabs;
+
+const RolesDetailPage = () => {
+  const { fqn } = useParams<{ fqn: string }>();
+
+  const [role, setRole] = useState<Role>({} as Role);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  const breadcrumb = useMemo(
+    () => [
+      {
+        name: 'Roles',
+        url: getSettingPath(
+          GlobalSettingsMenuCategory.ACCESS,
+          GlobalSettingOptions.ROLES
+        ),
+      },
+      {
+        name: fqn,
+        url: '',
+      },
+    ],
+    [fqn]
+  );
+
+  const fetchRole = async () => {
+    setLoading(true);
+    try {
+      const data = await getRoleByName(fqn, 'policies,teams,users');
+      setRole(data ?? ({} as Role));
```

Comment:
error handling here, and handle it wherever it is missing

Reply:
Same here, if API is successful then there should be some response, so we don't need to add extra checks